backend/.env

```
@@ -7,3 +7,5 @@ JWT_REFRESH_KEY=b87a8dfb2e8ddc23d8c5a8ea1f2eb8e13a39952d7ca0216348fa5e57b01c57ef
 
 ACCESS_TOKEN_EXPIRES_MINUTES=15
 REFRESH_TOKEN_EXPIRES_DAYS=7
+API_KEY=eyJvcmciOiI1YjNjZTM1OTc4NTExMTAwMDFjZjYyNDgiLCJpZCI6ImVjOTEyOGYyOWQ3MTQ2Mjk4ODRlNGEwMDBjYmNjMDkyIiwiaCI6Im11cm11cjY0In0=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The API_KEY key should go before the REFRESH_TOKEN_EXPIRES_DAYS key

(UnorderedKey)